wp-content/plugins/click-guide/options/action/saveWelcomeMessage.php

```diff
@@ -2,6 +2,13 @@
 
 	if(isset($_POST['submit'])){
 
+		// save status
+		if( isset( $_POST['clickguide_status'] ) and !empty( $_POST['clickguide_status'] ) ) {
```

Comment:
You don't need to check for `isset` since `empty` already does that (http://stackoverflow.com/a/4559976/2225200)